src/pages/index.js

```diff
@@ -13,28 +14,88 @@ import "./index.css";
 
 const profilePencilBtn = document.querySelector("#profile-pencil-btn");
 const profilePencilModal = document.querySelector("#profile-pencil-modal");
+const profileAvatarModal = document.querySelector("#profile-avatar-modal");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector("#profile-description-input");
+const profileAvatarImage = document.querySelector(".profile__image")
+const profileAvatarEditBtn = document.querySelector("#profile-image-pencil-btn");
 
 // Add Card Elements
 
 const profileAddBtn = document.querySelector("#profile-add-btn");
 const addCardModal = document.querySelector("#add-card-modal");
 
+// Api Class
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "5ed97d95-558f-499d-9d81-4e06bbe8932c",
+    "Content-Type": "application/json"
+  }
+}); 
+
+// UserInfo 
+
+const userInfo = new UserInfo({
+  title: '.profile__title',
+  description: '.profile__description',
+  avatar: '.profile__image'
+});
+
 // Popup Image Preview
 
 const imagePopup = new PopupWithImage("#preview-image-modal");
 
 imagePopup.setEventListeners();
 
+// Card Functions Handlers
+
 function handleCardClick(link, name) {
   imagePopup.open({link, name});
 }
 
+function handleLikeClick(card) {
+  if (card.isLiked()) {
+    api
+      .dislikeCard(card._id)
+      .then(() => {
+        card.setCardLike(false)
+      })
+      .catch((err) => {
+        console.error(`Error: ${err}`)
+      })
+  } else {
+    api
+      .likeCard(card._id)
+      .then(() => {
+        card.setCardLike(true)
+      })
+      .catch((err) => {
+        console.error(`Error: ${err}`)
+      })
+  }
+}
+
+function handleDeleteClick(card) {
+  deleteConfirmationCardPopup.open();
+  deleteConfirmationCardPopup.setConfirmationCallback(() => {
+    api
+      .deleteCard(card._id)
+      .then(() => {
+        card.handleDeleteCard();
+      })
+      .catch((err) => {
+        console.error(`Error: ${err}`)
+      })
+  })
+}
+
+
 // Create Card Function
 
 function createCard(cardData) {
-  const cardElementData = new Card(cardData, "#card-template", handleCardClick);
+  const cardElementData = new Card(cardData, "#card-template", handleCardClick, handleLikeClick, handleDeleteClick);
   return cardElementData.getView();
 }
 
```

Comment:
```js

const defautlFormConfig = {
  formSelector: ".form",
  inputSelector: ".form__input",
  submitButtonSelector: ".modal__save-button",
  inactiveButtonClass: "modal__save-button_disabled",
  inputErrorClass: "form__input-error",
  errorClass: "form__error_visible"
}
```

From the 8th sprint all constants ( `defautlFormConfig `) should be in file `constants.js` in folder `utils` to clean up the main file.

Reply:
Moved to constants.js file and imported in index.js file